composer.json

```
@@ -68,12 +73,32 @@
 	"autoload-dev": {
 		"psr-4": { "Imagify\\Tests\\": "Tests/"	}
 	},
+	"extra": {
+		"mozart": {
+			"dep_namespace": "ImagifyPlugin\\Dependencies\\",
```

```suggestion
			"dep_namespace": "Imagify\\Dependencies\\",
```
Let's use the same namespace as our internals for consistency